utils/common/src/main/java/brooklyn/config/ConfigInheritance.java

```diff
@@ -18,9 +18,11 @@
  */
 package brooklyn.config;
 
+import java.io.Serializable;
+
 import com.google.common.annotations.Beta;
 
-public abstract class ConfigInheritance {
+public abstract class ConfigInheritance implements Serializable {
```

Comment:
I'll just ignore the warnings, since it's not needed for actual serialisation here.